fsm/ramping.c

```diff
@@ -41,9 +41,16 @@ inline void set_level_aux_leds(uint8_t level) {
 #include "anduril/aux-leds.h"  // for rgb_led_voltage_readout()
 inline void set_level_aux_rgb_leds(uint8_t level) {
     if (! go_to_standby) {
+        #ifdef USE_CONFIGURABLE_RGB_VOLTAGE_LEVELS
+        if ((level > 0) && (actual_level > cfg.use_aux_rgb_leds_while_on_min_level)){
+            rgb_led_voltage_readout(level > cfg.use_aux_rgb_leds_while_on);
+        }
+        #else
         if (level > 0) {
             rgb_led_voltage_readout(level > USE_AUX_RGB_LEDS_WHILE_ON);
-        } else {
+        }
+        #endif
+        else {
```

Comment:
this looks bogus to me?

Reply:
sorry it's fine my editor was glitching up

Reply:
No problem, I do that sort of thing all the time :P

Reply:
> No problem, I do that sort of thing all the time :P

brute [merged](https://github.com/gretel/anduril/commits/trunk/) most of the open prs in my fork and all werking fine including this.